WalletWasabi.Fluent/Controls/ContentArea.axaml

```diff
@@ -16,10 +15,10 @@
   <Style Selector="c|ContentArea">
     <Setter Property="Template">
       <ControlTemplate>
-        <DockPanel Margin="20 36 20 0">
-          <StackPanel DockPanel.Dock="Top" Spacing="10" Margin="0 0 0 40">
+        <DockPanel Name="PART_DockPanel" >
+          <StackPanel Name="PART_StackPanel" DockPanel.Dock="Top" Spacing="10">
             <ContentPresenter Name="PART_TitlePresenter" Content="{TemplateBinding Title}" />
-            <ContentPresenter Name="PART_CaptionPresenter" Content="{TemplateBinding Caption}" />
+            <TextBlock Name="PART_CaptionPresenter" Text="{TemplateBinding Caption}" />
```

Comment:
we need to keep this as contentpresenter... id like to use this space later on for adding controls on the right hand side...

you can have a selector like "ContentPresenter TextBlock" and set wrapping on that... it means any textblock that is child of the contentpresenter (or sub child)

Reply:
As we discuss on Slack I cannot set the ContentPresenter TextBlock TextWrapping property via style. Need to debug this in Avalonia.